chapters/functions.tex

```diff
@@ -2365,10 +2365,29 @@ \subsection{Annotations for External Functions}\label{annotations-for-external-l
   The {\lstinline!annotation(Library={"libraryName1", "libraryName2"})!}, used by the linker to include the library files where the compiled external function is available and additional libraries used to implement it.
   For shared libraries it is recommended to include all non-system libraries in this list.
 \item
-  The {\lstinline!annotation(Include="includeDirective")!}\annotationindex{Include}, used to include source files needed for calling the external function in the code generated by the Modelica compiler.
+  The {\lstinline!annotation(Include="insertedCode")!}\annotationindex{Include}, used to insert function prototypes or definitions needed for calling the external function in the code generated by the Modelica compiler.
+  When generating a call to the external function, the \lstinline!"insertedCode"! shall be present at the top level somewhere before the point of the call (similar to where include directives are typically placed).
+  The \lstinline!Include! annotation shall be used in such a way that each external function can be handled in a separate translation unit.
+  In particular, different external functions must not have \lstinline!Include! annotations providing exported definitions of the same funciton symbol to avoid linking errors.
```

Comment:
```suggestion
  In particular, different external functions must not have \lstinline!Include! annotations providing exported definitions of the same function symbol to avoid linking errors.
```